### PR TITLE
feat: Horizontal card list read more

### PR DESCRIPTION
**Pull request by vinnyjth**
### What does this PR do, or why is it needed?

There's a storybook story to test the buttons, or you can use the linked setup and this PR: https://github.com/ApollosProject/apollos-templates/pull/197 

to test everything

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-ui-connected/src/FeaturesFeedConnected/FeaturesFeedConnected.js [hunk 1]
```diff
@@ -30,6 +30,12 @@ export const ACTION_MAP = {
   OPEN_URL: ({ openUrl, relatedNode }) => {
     openUrl(relatedNode.url);
   },
+  OPEN_CHANNEL: ({ relatedNode, navigation }) => {
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/FeaturesFeedConnected/FeaturesFeedConnected.js:33]: This has been implemented in at least one other client app btw :)

**Comment by redreceipt** [comment]
Any reason you switched from "View More" to "Read More"? I think I like the former better

**Comment by vinnyjth** [comment]
@redreceipt It can be whatever you want, it's configured in the `config.yml`

**Comment by redreceipt** [comment]
> @redreceipt It can be whatever you want, it's configured in the `config.yml`

oh nice

**Review by redreceipt — APPROVED** [review]

**Pull request merged**